react-search-component/my-app/src/SearchableList Components/SearchBar.tsx

```diff
@@ -0,0 +1,25 @@
+import { ChangeEvent } from 'react';
+
+type SearchBarProps = {
```

Comment:
It's just a matter of preference, but since this type isn't exported, I prefer to just call it Props. Less typing!

Reply:
updated